views/home.py

```diff
@@ -12,6 +12,16 @@
 import orm
 import subprocess
 
+@app.route('/register',methods=['POST'])
```

Comment:
@pontikos, what's the plan about this - do we want it to also send you an email so you can approve them?